backend/curriculum_tracking/tests/factories.py

```diff
@@ -238,6 +238,7 @@ class Meta:
     recruit_project = factory.SubFactory(
         RecruitProjectFactory, start_time=timezone.now()
     )
+    topic_progress = factory.SubFactory(TopicProgressFactory)
```

Comment:
A single card cannot be linked to both a topic and a project. This factory now creates an invalid object.